### PR TITLE
Added order collection and updated existing collections

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

**Review by himank — APPROVED** [review]

File: src/main/java/com/tigrisdata/starter/spring/TigrisDBInitializer.java [hunk 1]
```diff
@@ -17,47 +17,60 @@
...
+        log.info("creating db {}", dbName);
+        tigrisDBClient.createDatabase(dbName, new DatabaseOptions());
+
+        TigrisDatabase tigrisDatabase = tigrisDBClient.getDatabase(dbName);
+        createCollections(tigrisDatabase);
+        log.info("Finished initializing TigrisDB");
+    }
+
+    private void createCollections(TigrisDatabase tigrisDatabase) throws IOException {
+        for (File schemaFile : new File("src/main/resources/tigrisdb-schema/")
```
Comment (ovaistariq) [inline src/main/java/com/tigrisdata/starter/spring/TigrisDBInitializer.java:53]: There is a way to create all the collections transactionally in a single transaction. Let's do that.

File: src/main/resources/tigrisdb-schema/order.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    "user_id": {
+      "description": "The identifier of the user that placed the order",
+      "type": "bigint"
+    },
+    "order_total": {
+      "description": "The total cost of the order",
+      "type": "double"
+    },
+    "product_item": {
```
Comment (ovaistariq) [inline src/main/resources/tigrisdb-schema/order.json:17]: Nit: product_items

Comment (ovaistariq) [reply src/main/resources/tigrisdb-schema/order.json:17]: Don't forget to update the example schema in the docs

Comment (JigarJoshi) [reply src/main/resources/tigrisdb-schema/order.json:17]: Thanks for catching it.  I will update this PR after I merge https://github.com/tigrisdata/tigrisdb-client-java/pull/37 and publish the client.

**Review by ovaistariq — APPROVED** [review]

**Review by himank — APPROVED** [review]

**Pull request merged**